msaad.go

```diff
@@ -523,6 +525,56 @@ func (m *MSAAD) syncRoles(roleGroups *cachedRoleGroups, aadUser *msaadUser, inte
 	groupsChanged := false
 	userHasAnyIMQSPermission := false
 
+	// identify unmapped groups
+	removeIDs, _ := DecodePermit(make([]byte, 0))
+	allowedIDs, _ := DecodePermit(make([]byte, 0))
+
+	m.parent.Log.Debugf("MSAAD empty role arrays constructed")
+
+	// get all mapped group ids
+	for _, internalGroupName := range m.Config.RoleToGroup {
+		m.parent.Log.Debugf("MSAAD checking all roles: %v", internalGroupName)
```

Comment:
I think this class of logs should not be allowed to bloat the configs, unless we explicitly want them. So I will just let you know all of the logs that I am sure should be masked behind verbose config.
```suggestion
		if m.parent.MSAAD.Config.Verbose {
		    m.parent.Log.Debugf("MSAAD checking all roles: %v", internalGroupName)
		}
```